src/components/Analysis/Options.tsx

```diff
@@ -0,0 +1,27 @@
+import { OptionsProps } from "../../pages/Analysis/Analysis.types";
+
+export const Options = ({ options }: OptionsProps) => {
+  const getOptionStyle = (
+    isOptionSelected: Boolean,
+    isOptionRight: Boolean
+  ): string => {
+    if (isOptionRight) {
+      return "option-btn option-correct";
+    }
+    if (isOptionSelected && !isOptionRight) {
+      return "option-btn option-wrong";
+    }
+    return "option-btn option-unselected";
+  };
```

Comment:
Leave a new line before the return of the function component so as to separate logic from the view and improve code readability.